dcpy/connectors/data_library.py

```diff
@@ -0,0 +1,59 @@
+from sqlalchemy import text, update, insert
+from sqlalchemy.schema import Table, MetaData
+
+from dcpy import BUILD_ENGINE_RAW, build_engine
+from dcpy import DCPY_ROOT_PATH
+from dcpy.connectors import psql
+from dcpy.connectors.edm import recipes
+
+LIBRARY_DEFAULT_PATH = DCPY_ROOT_PATH.parent / ".library"
+
+
+def create_source_data_table():
+    """Create the source_data_versions table"""
+    with build_engine.begin() as con:
+        con.execute(
+            text(
+                """DROP TABLE IF EXISTS source_data_versions;
+                CREATE TABLE source_data_versions (
+                schema_name character varying,
+                v character varying);"""
+            )
+        )
+
+
+def import_recipe(
+    recipe_name: str,
+    *,
```

Comment:
@damonmcc (and @athursland and @DeaBardhoshi ) mentioned this to Finn, but a convention I like in functions with many arguments is adding the `*`, which forces everything after to be a keyword argument. So I can't invoke this fn like
```
import_recipe('my_recipe_name', 'latest', 'my_path', False)
```
Instead when I want to override a default, I have to specify it 
```
import_recipe('my_recipe_name',  set_version=False)
```
Does wonders for readability in the implementation.